src/ui/components/shared/LoadingScreen.tsx

```diff
@@ -17,60 +18,77 @@ const Hoverboard = dynamic(() => import("./Hoverboard"), {
 export function LoadingScreenTemplate({ children }: { children?: ReactNode }) {
   const [hoverboardColor, setHoverboardColor] = useState(colorOptions[2]);
 
-  const changeHoverboardColor = () => {
+  const changeHoverboardColor = useCallback(() => {
     const randomIndex = Math.floor(Math.random() * colorOptions.length);
     setHoverboardColor(colorOptions[randomIndex]);
-  };
+  }, []);
 
   useEffect(() => {
-    const interval = setInterval(changeHoverboardColor, 5_000);
-    return () => clearInterval(interval);
-  }, [hoverboardColor]);
+    const timeoutId = setTimeout(changeHoverboardColor, 5000);
+    return () => clearTimeout(timeoutId);
+  }, [changeHoverboardColor]);
```

Comment:
I think there was a misunderstanding from my comment https://github.com/replayio/devtools/pull/9220#discussion_r1199032258

I meant do one or the other, not both. The code now will only change the color once. (I think you intended the color to change every 5 seconds?) Maybe I'm misunderstanding.